cirq-google/cirq_google/workflow/io.py

```diff
@@ -44,6 +44,23 @@ class ExecutableGroupResultFilesystemRecord:
 
     run_id: str
 
+    @classmethod
+    def from_json(
+        cls, *, run_id: str,  base_data_dir: str = "."
+    ) -> 'ExecutableGroupResultFilesystemRecord':
+        fn = f'{base_data_dir}/{run_id}/ExecutableGroupResultFilesystemRecord.json.gz'
+        egr_record = cirq.read_json_gzip(fn)
```

Comment:
Nice, I didn't realize we were also storing `ExecutableGroupResultFilesystemRecord` and therefore we can fetch the `runtime_configuration_path`, `shared_runtime_info_path` etc. by directly reading this json instead of hardcoding any paths here.